src/mcp_atlassian/utils/oauth_setup.py

```diff
@@ -238,12 +263,90 @@ def run_oauth_flow(args: OAuthSetupArgs) -> bool:
 
         if oauth_config.cloud_id:
             logger.info(f"Cloud ID: {oauth_config.cloud_id}")
-            logger.info("\nAdd the following to your .env file:")
+
+            # Print environment variable information more clearly
```

Comment:
Please add documentation to `SECURITY.md` explaining the security risk associated with printing the client secret (`ATLASSIAN_OAUTH_CLIENT_SECRET`) directly to the console output.

Reply:
Added